spec/models/loader/types.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
+
+describe Loader::Types do
+
+  let(:new_user) {  Loader::Types::User.new(id: 'alice') }
+
+  describe '.check_user_creation_allowed' do
+    context "Check user creation verification" do
+
+      it "should not allow user creation not under root" do
+        allow(ENV).to receive(:[]).with('CONJUR_ALLOW_USER_CREATION').and_return('false')
+        user = new_user()
```

Comment:
looks like `user` is never being used.  We can remove it.

Reply:
fixed